main.js

```diff
@@ -179,20 +192,39 @@ function startMultihost(__config) {
                     tools.decryptPhrase(obj.native.secret, _config.multihostService.password, secret =>
                         _startMultihost(_config, secret));
                 } else {
-                    logger.error(hostLogPrefix + ' Cannot start multihost discovery server: no system.config found (err:' + err + ')');
+                    logger.error(`${hostLogPrefix} Cannot start multihost discovery server: no system.config found (err:${err})`);
                 }
             });
         } else {
             _startMultihost(_config, false);
         }
 
+        if (!_config.multihostService.persist) {
+            mhTimer = setTimeout(async () => {
```

Comment:
Ok, so if someone enables discovery, then stops iobroker and restarts it will be ena´led 15 mins again, or?

Reply:
Yes.

Reply:
Do we want it that way? Makes that sense?

Reply:
It only is the case if user restarts before timeout callback is executed (15 mins). Alternative would be to disable on iob stop but this could also confuse the user. Or also saving the activation timestamp and checking on start.. but I thought it would be ok like this.

Reply:
Not nice, but ok ...

Reply:
I am open to alternatives, but all alternatives which came to my mind were also not nice.